build.rs

```diff
@@ -0,0 +1,93 @@
+#[macro_use] extern crate error_chain;
+extern crate curl;
```

Comment:
It's not based on CURL, it's written in Rust completely. It's dependencies are bare Rust too.